app/assets/javascripts/solidus_paypal_braintree/constants.js

```diff
@@ -0,0 +1,50 @@
+SolidusPaypalBraintree = {
+  APPLY_PAY_API_VERSION: 1,
+
+  config: {
+    paths: {
+      clientTokens: Spree.pathFor('solidus_paypal_braintree/client_token'),
```

Comment:
This leads to `Spree is not defined` errors when loading this file asynchronously (what we do on our production setup). I propose to introduce a `SolidusPaypalBraintree.init()` function that gets invoked in `checkout.js` after Dom ready (what we have overridden, due to our asynchronous loading).

Will provide a PR with a fix, after this gets merged, as this is not an issue with this implementation here.